# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
#
# Николай Семенов, 2011.
# Alexey Ivanov <alexey.ivanes@gmail.com>, 2011.
msgid ""
msgstr ""
"Project-Id-Version: 1.0\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2013-04-05 17:38+0300\n"
"PO-Revision-Date: 2011-01-19 01:37+0300\n"
"Last-Translator: Alexey Ivanov <alexey.ivanes@gmail.com>\n"
"Language-Team: Russian <kde-russian@lists.kde.ru>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
"%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
"X-Generator: Lokalize 1.1\n"

#: gui/ScrInstall.py:161
#, python-format
msgid "Installing <b>%(name)s</b> -- %(summary)s"
msgstr "Установка <b>%(name)s</b><br>%(summary)s"

#: gui/ScrInstall.py:170
#, python-format
msgid "Configuring <b>%s</b>"
msgstr "Настройка <b>%s</b>"

#: gui/ScrInstall.py:195 gui/ScrPartitionAuto.py:44
#: gui/ScrPartitionManual.py:138 gui/ScrCheckCD.py:96 gui/YaliWindow.py:212
#: gui/YaliWindow.py:217 gui/YaliWindow.py:222
msgid "Warning"
msgstr "Предупреждение"

#: gui/ScrInstall.py:196
#, fuzzy, python-format
msgid ""
"Following error occured while installing packages:<b>%s</b><br><br>Do you "
"want to retry?"
msgstr ""
"Произошла ошибка во время установки пакетов:\n"
"<b>%s</b>"

#: gui/ScrInstall.py:202 gui/YaliDialog.py:218 gui/YaliDialog.py:234
#: gui/ScrSummary.py:297 storage/storageBackendHelpers.py:233
#: storage/storageBackendHelpers.py:254 storage/storageBackendHelpers.py:304
msgid "Yes"
msgstr "Да"

#: gui/ScrInstall.py:202 gui/YaliDialog.py:220 gui/YaliDialog.py:235
#: gui/ScrSummary.py:297 storage/storageBackendHelpers.py:233
#: storage/storageBackendHelpers.py:254 storage/storageBackendHelpers.py:304
msgid "No"
msgstr "Нет"

#: gui/ScrInstall.py:258
#, python-format
msgid ""
"An error occured during the installation of packages.\n"
"This may be caused by a corrupted installation medium error:\n"
"%s\n"
msgstr ""
"Возникла ошибка в процессе установки пакетов.\n"
"\n"
"Причиной этому может служить поврежденный установочный носитель.\n"
"\n"
"Ошибка:\n"
"%s\n"

#: gui/runner.py:117
msgid "Error!"
msgstr "Ошибка!"

#: gui/runner.py:118
#, python-format
msgid ""
"An error occurred when attempting to load an installer interface component.\n"
"\n"
"className = %s.Widget"
msgstr ""
"Произошла ошибка при попытке загрузить компонент интерфейса установки.\n"
"\n"
"className = %s.Widget"

#: gui/runner.py:122 gui/ScrRescuePassword.py:123 gui/ScrRescue.py:95
#: gui/YaliWindow.py:385 storage/storageset.py:388
msgid "Exit"
msgstr "Выход"

#: gui/runner.py:122
msgid "Retry"
msgstr "Повторить"

#: gui/ScrPartitionAuto.py:45
#, python-format
msgid ""
"No partitions are available to resize. Only physical\n"
"partitions which size is greater than %s MB can be resized."
msgstr ""
"Нет разделов, доступных для изменения их размера. Только физические\n"
"разделы, размер которых более %s MB могут быть доступны для изменения "
"размера."

#: gui/ScrPartitionAuto.py:60
#, fuzzy
msgid "Use All Disks"
msgstr "Использовать все пространство."

#: gui/ScrKeyboard.py:73
msgid "Use Alt-Shift to toggle between alternative keyboard layouts"
msgstr ""
"Используйте комбинацию клавиш Alt-Shift для переключения альтернативных "
"раскладок клавиатуры"

#: gui/ScrPartitionManual.py:94
#, fuzzy, python-format
msgid ""
"The partitioning scheme you requested caused the following critical errors."
"You must correct these errors before you continue your installation of %s."
msgstr ""
"Схема разметки, предлагаемая вами, вызвала следующие критические ошибки. Вы "
"должны исправить эти ошибки, перед тем как продолжить установку %s."

#: gui/ScrPartitionManual.py:101
msgid "Partitioning Errors"
msgstr "Ошибки разметки дисков"

#: gui/ScrPartitionManual.py:106
#, fuzzy
msgid ""
"The partitioning scheme you requested generated the following warnings. "
"Would you like to continue with your requested partitioning scheme?"
msgstr ""
"Схема разметки содержит предупреждения. Вы хотите продолжить с данной схемой "
"разметки?"

#: gui/ScrPartitionManual.py:112
msgid "Partitioning Warnings"
msgstr "Предупреждения разметки дисков"

#: gui/ScrPartitionManual.py:114 gui/ScrPartitionManual.py:140
#: gui/YaliDialog.py:218 storage/__init__.py:164
msgid "Ok"
msgstr "Ok"

#: gui/ScrPartitionManual.py:114 gui/ScrPartitionManual.py:130
#: gui/ScrPartitionManual.py:140 gui/YaliDialog.py:220
#: gui/Ui/partitionshrinkwidget.py:103 gui/Ui/partresize.py:103
#: gui/Ui/setupuserswidget.py:513 gui/lvm_gui.py:418 gui/lvm_gui.py:465
#: gui/lvm_gui.py:552 storage/storageBackendHelpers.py:92
#: storage/storageBackendHelpers.py:296 storage/__init__.py:930
msgid "Cancel"
msgstr "Отменить"

#: gui/ScrPartitionManual.py:120
#, fuzzy
msgid ""
"The following pre-existing devices have been selected to be formatted, "
"destroying all data."
msgstr ""
"Следующие уже существующие устройства\n"
"были выбраны для форматирования. При этом\n"
"будут уничтожены все данные."

#: gui/ScrPartitionManual.py:128
msgid "Format Warnings"
msgstr "Предупреждения форматирования"

#: gui/ScrPartitionManual.py:130 gui/Ui/manualpartwidget.py:197
#: gui/Ui/raid.py:117 gui/Ui/partition.py:157 gui/Ui/logicalvolume.py:159
#: storage/storageset.py:387
msgid "Format"
msgstr "Форматировать"

#: gui/ScrPartitionManual.py:138
msgid "All Changes that you made will be removed"
msgstr "Все изменения, сделанные вами, будут удалены"

#: gui/ScrPartitionManual.py:148
msgid "Standard"
msgstr "Стандартный"

#: gui/ScrPartitionManual.py:149
msgid "LVM"
msgstr "LVM"

#: gui/ScrPartitionManual.py:150
msgid "RAID"
msgstr "RAID"

#: gui/ScrPartitionManual.py:152
msgid "Partition"
msgstr "Раздел"

#: gui/ScrPartitionManual.py:153 gui/Ui/createdevicewidget.py:144
msgid "General purpose of partition creation"
msgstr "Общее назначение создания разделов"

#: gui/ScrPartitionManual.py:155 gui/Ui/createdevicewidget.py:149
msgid "Physical Volume"
msgstr "Физический том"

#: gui/ScrPartitionManual.py:156 gui/Ui/createdevicewidget.py:150
msgid "Create LVM formatted partition"
msgstr "LVM"

#: gui/ScrPartitionManual.py:158 gui/Ui/createdevicewidget.py:145
msgid "Volume Group"
msgstr "Группа томов"

#: gui/ScrPartitionManual.py:159 gui/Ui/createdevicewidget.py:146
msgid "Requires at least 1 free LVM formatted partition"
msgstr "Требуется по меньшей мере 1 свободный LVM отформатированный раздел"

#: gui/ScrPartitionManual.py:161 gui/Ui/createdevicewidget.py:147
msgid "Logical Volume"
msgstr "Логический том"

#: gui/ScrPartitionManual.py:162 gui/Ui/createdevicewidget.py:148
msgid "Create Logical Volume on selected Volume Group"
msgstr "Создать логическую группу в выбранной группе по размеру"

#: gui/ScrPartitionManual.py:164
msgid "Member"
msgstr "Участник"

#: gui/ScrPartitionManual.py:165 gui/Ui/createdevicewidget.py:152
msgid "Create Raid formatted partition"
msgstr "Создать отформатированный раздел RAID"

#: gui/ScrPartitionManual.py:167
msgid "Array"
msgstr "Массив"

#: gui/ScrPartitionManual.py:168 gui/Ui/createdevicewidget.py:154
msgid "Requires at least 2 free Raid formatted partition"
msgstr "Требует как минимум 2 свободных отформатированных разделов RAID"

#: gui/ScrPartitionManual.py:231
msgid "Volume Groups"
msgstr "Группы томов"

#: gui/ScrPartitionManual.py:244 gui/ScrPartitionManual.py:320
msgid "Free"
msgstr "Свободно"

#: gui/ScrPartitionManual.py:253
msgid "Raid Arrays"
msgstr "Raid массивы"

#: gui/ScrPartitionManual.py:268
msgid "Hard Drives"
msgstr "Жесткие диски"

#: gui/ScrPartitionManual.py:297
msgid "Can't handle more than one extended partition per disk"
msgstr "Нельзя иметь больше чем один расширенный раздел на диске"

#: gui/ScrPartitionManual.py:304
msgid "Crossed logical partition before extended"
msgstr "Пересечение логических разделов перед расширением."

#: gui/ScrPartitionManual.py:325
msgid "Extended"
msgstr "Расширенный"

#: gui/ScrPartitionManual.py:351 storage/partitioning.py:1648
#: storage/partitioning.py:1680
msgid "Error Partitioning"
msgstr "Ошибка разметки"

#: gui/ScrPartitionManual.py:352
#, python-format
msgid "Could not allocate requested partitions: %s."
msgstr "Не удалось разместить требуемые разделы: %s."

#: gui/ScrPartitionManual.py:356
msgid "Warning Partitioning"
msgstr "Предупреждение разметки"

#: gui/ScrPartitionManual.py:357
#, python-format
msgid "Warning: %s."
msgstr "Предупреждение: %s."

#: gui/ScrPartitionManual.py:358 storage/storageBackendHelpers.py:319
msgid "Modify Partition"
msgstr "Изменить раздел"

#: gui/ScrPartitionManual.py:358 gui/ScrRescuePassword.py:123
#: gui/ScrRescue.py:95 gui/lvm_gui.py:418 gui/lvm_gui.py:465
#: storage/storageset.py:336
msgid "Continue"
msgstr "Продолжить"

#: gui/ScrPartitionManual.py:472
msgid "Partition Selection Warning"
msgstr "Предупреждение выбора раздела"

#: gui/ScrPartitionManual.py:473
msgid "Please select free physical partition to create new device."
msgstr ""
"Пожалуйста выберите свободный физический раздел для создания нового "
"устройства."

#: gui/ScrPartitionManual.py:483
msgid "Unable To Edit"
msgstr "Редактирование невозможно"

#: gui/ScrPartitionManual.py:484
#, python-format
msgid ""
"You cannot edit this device:\n"
"\n"
"%s"
msgstr ""
"Вы не можете редактировать это устройство:\n"
"\n"
"%s"

#: gui/ScrKahyaCheck.py:78
msgid "No storage device found."
msgstr "Устройств хранения данных не найдено."

#: gui/ScrRescuePassword.py:99 gui/ScrUsers.py:214
msgid "Don't use your user name or name as a password"
msgstr "Не используйте ваше имя пользователя или логин как пароль"

#: gui/ScrRescuePassword.py:102 gui/ScrUsers.py:217
msgid "Passwords do not match"
msgstr "Пароли не совпадают"

#: gui/ScrRescuePassword.py:105 gui/ScrUsers.py:220
msgid "Password is too short"
msgstr "Пароль слишком короткий"

#: gui/ScrRescuePassword.py:120 gui/ScrRescue.py:92
msgid "Cannot Rescue"
msgstr ""

#: gui/ScrRescuePassword.py:121 gui/ScrRescue.py:93
#, fuzzy
msgid "Your current installation cannot be rescued."
msgstr "Выбрана автоматическая установка."

#: gui/ScrRescuePassword.py:145
msgid "Super User"
msgstr "Администратор"

#: gui/YaliDialog.py:240
msgid "Don't ask again"
msgstr "Больше не спрашивать"

#: gui/YaliDialog.py:245
msgid "Do you want to continue?"
msgstr "Вы хотите продолжить?"

#: gui/YaliDialog.py:260
msgid "Information"
msgstr "Информация"

#: gui/YaliDialog.py:262
msgid "OK"
msgstr "ОК"

#: gui/YaliDialog.py:453
msgid "Error reporting"
msgstr "Сообщить об ошибке"

#: gui/ScrCheckCD.py:62 gui/Ui/checkcdwidget.py:213
msgid "Validate"
msgstr "Проверка"

#: gui/ScrCheckCD.py:68
msgid "Starting validation..."
msgstr "Запуск проверки..."

#: gui/ScrCheckCD.py:82
msgid "Package validation is in progress. Please wait until it is completed."
msgstr ""

#: gui/ScrCheckCD.py:89
#, python-format
msgid "Validating %s"
msgstr "Проверка %s"

#: gui/ScrCheckCD.py:97
#, python-format
msgid ""
"Validation of %s package failed.Please remaster your installation medium "
"andreboot."
msgstr ""
"Проверка пакета %s завершилась с ошибкой. Пожалуйста, замените ваш "
"установочный диск и перезагрузитесь."

#: gui/ScrCheckCD.py:101
msgid "Skip Validation"
msgstr "Пропустить проверку"

#: gui/ScrCheckCD.py:101
msgid "Skip Package"
msgstr "Пропустить пакет"

#: gui/ScrCheckCD.py:101 gui/Ui/exception.py:73 storage/__init__.py:930
msgid "Reboot"
msgstr "Перезагрузить"

#: gui/ScrCheckCD.py:115
msgid ""
"<font color=\"#FFF\"><b>Validation succeeded. You can proceed with the "
"installation.</b></font>"
msgstr ""
"<font color=\"#FFF\"><b>Проверка завершилась успешно. Теперь Вы можете "
"продолжить установку.</b></font>"

#: gui/ScrCheckCD.py:127
#, fuzzy
msgid "Package validation is finished."
msgstr "Настройка установки пакетов"

#: gui/shrink_gui.py:23
msgid "Partitions to Shrink"
msgstr "Разделы для уменьшения"

#: gui/shrink_gui.py:47
msgid "Resize FileSystem Error"
msgstr "Ошибка изменения размера в файловой системе"

#: gui/shrink_gui.py:48
#, python-format
msgid "%(device)s: %(msg)s"
msgstr "%(device)s: %(msg)s"

#: gui/shrink_gui.py:56
msgid "Resize Device Error"
msgstr "Ошибка изменения разрамера на устройстве"

#: gui/shrink_gui.py:57
#, python-format
msgid "%(name)s: %(msg)s"
msgstr "%(name)s: %(msg)s"

#: gui/raid_gui.py:26 gui/raid_gui.py:96 gui/raid_gui.py:124
msgid "Invalid Raid Members"
msgstr "Ошибочные члены Raid"

#: gui/raid_gui.py:27
msgid ""
"At least two unused software RAID partitions are needed to create a RAID "
"device.\n"
"\n"
"First create at least two partitions of type \"software RAID\", and then "
"select the \"RAID\" option again."
msgstr ""
"Требуется по крайней мере два неиспользуемых раздела RAID для создания "
"устройства RAID.\n"
"\n"
"Сначала создайте по крайней мере два раздела типа \"программный RAID\" и "
"выберете опцию \"RAID\" снова."

#: gui/raid_gui.py:37
msgid "Make RAID Device"
msgstr "Создать RAID устройство"

#: gui/raid_gui.py:40
#, python-format
msgid "Edit RAID Device: %s"
msgstr "Редактировать RAID устройство: %s"

#: gui/raid_gui.py:42
msgid "Edit RAID Device"
msgstr "Редактировать RAID устройство"

#: gui/raid_gui.py:68 gui/partition_gui.py:65 gui/lvm_gui.py:613
msgid "Mount Point Error"
msgstr "Ошибка монтирования"

#: gui/raid_gui.py:82 gui/partition_gui.py:79 gui/lvm_gui.py:644
msgid "Mount point in use"
msgstr "Точка монтирования уже используется"

#: gui/raid_gui.py:83 gui/partition_gui.py:80 gui/lvm_gui.py:645
#, python-format
msgid "The mount point \"%s\" is in use. Please pick another."
msgstr ""
"Точка монтирования \"%s\" уже используется. Пожалуйста, укажите другую."

#: gui/raid_gui.py:97 storage/devices/raidarray.py:60
#, python-format
msgid "A RAID%(level)d set requires at least %(min_member)d member"
msgstr "RAID%(level)d набор требует хотя бы %(min_member)d элемент"

#: gui/ScrLicense.py:77
msgid "License text could not be found."
msgstr "Текст лицензии не может быть найден."

#: gui/ScrRescue.py:54
#, fuzzy
msgid "Mount failed"
msgstr "Не удалось установить соединение"

#: gui/ScrRescue.py:55
#, python-format
msgid ""
"The following error occurred when mounting the file systems listed in /etc/"
"fstab.  Please fix this problem and try to rescue again.\n"
"%s"
msgstr ""

#: gui/ScrDateTime.py:169
msgid "Adjusting time settings"
msgstr "Регулировка настроек времени"

#: gui/ScrDateTime.py:208 gui/ScrAdmin.py:96
msgid "Storage Devices initialising..."
msgstr "Определение устройств хранения данных..."

#: gui/ScrAdmin.py:154
msgid "Password is too short."
msgstr "Пароль слишком короткий."

#: gui/ScrAdmin.py:162
msgid "Passwords do not match."
msgstr "Пароли не совпадают."

#: gui/ScrAdmin.py:167
msgid "Do not use your username as your password."
msgstr "Не используйте свое имя пользователя в качестве пароля."

#: gui/ScrAdmin.py:177
msgid "Hostname cannot be longer than 64 characters."
msgstr "Имя хоста не может быть длиннее 64 символов."

#: gui/ScrAdmin.py:190
msgid "Hostname contains invalid characters."
msgstr "Имя хоста содержит недопустимые символы."

#: gui/storageGuiHelpers.py:71 gui/lvm_gui.py:402 gui/lvm_gui.py:432
#: gui/lvm_gui.py:445 gui/lvm_gui.py:480 gui/lvm_gui.py:696 gui/lvm_gui.py:718
msgid "Not enough space"
msgstr "Недостаточно места"

#: gui/storageGuiHelpers.py:72
msgid ""
"You cannot remove this physical volume because\n"
"otherwise the volume group will be too small to\n"
"hold the currently defined logical volumes."
msgstr ""
"Вы не можете удалить этот физический том\n"
"потому что тогда группа томов станет слишком\n"
"малой для поддержки текущих логических томов."

#: gui/ScrGoodbye.py:37
msgid "Running post-install operations..."
msgstr "Запуск после установочных операций..."

#: gui/ScrGoodbye.py:49
msgid "Press <b>Restart</b> to restart the computer."
msgstr "Нажмите <b>Перезагрузка</b> чтобы перезагрузить компьютер."

#: gui/ScrGoodbye.py:49 gui/ScrSummary.py:46
msgid "Restart"
msgstr "Перезапуск"

#: gui/ScrGoodbye.py:50
msgid "<b>Please wait while restarting...</b>"
msgstr "<b>Пожалуйста, подождите, идет перезагрузка...</b>"

#: gui/ScrGoodbye.py:64
msgid "Setting timezone..."
msgstr "Настройка часового пояса..."

#: gui/ScrGoodbye.py:65
msgid "Migrating Xorg configuration..."
msgstr "Перенос конфигурации Xorg"

#: gui/ScrGoodbye.py:66
msgid "Copying repository index..."
msgstr "Копирование индекса репозитория..."

#: gui/ScrGoodbye.py:69
msgid "Setting hostname..."
msgstr "Настройка имя хоста..."

#: gui/ScrGoodbye.py:70
msgid "Setting root password..."
msgstr "Настройка пароля администрация..."

#: gui/ScrGoodbye.py:73
#, fuzzy
msgid "Resetting user password..."
msgstr "Настройка пароля администрация..."

#: gui/ScrGoodbye.py:76
msgid "Setup First-Boot..."
msgstr "Установка первой загрузки..."

#: gui/ScrGoodbye.py:79
#, fuzzy
msgid "Teardown First-Boot..."
msgstr "Установка первой загрузки..."

#: gui/ScrGoodbye.py:83
msgid "Adding users..."
msgstr "Добавление пользователей..."

#: gui/ScrGoodbye.py:88
msgid "Installing bootloader..."
msgstr "Установка загрузчика..."

#: gui/ScrGoodbye.py:89
#, fuzzy
msgid "Writing bootloader config..."
msgstr "Запись загрузчика..."

#: gui/ScrGoodbye.py:90 gui/ScrGoodbye.py:94
msgid "Stopping to D-Bus..."
msgstr "Остановка D-Bus..."

#: gui/ScrGoodbye.py:91 gui/ScrGoodbye.py:95
msgid "Teardown storage subsystem..."
msgstr ""

#: gui/ScrRescuePisi.py:37
msgid "Starting D-Bus..."
msgstr "Запуск D-Bus..."

#: gui/ScrRescuePisi.py:38
msgid "Connecting to D-Bus..."
msgstr "Подключение к D-Bus ..."

#: gui/ScrRescuePisi.py:39
msgid "Fetching history..."
msgstr "Получение истории ..."

#: gui/ScrRescuePisi.py:69
msgid "Please Wait..."
msgstr "Пожалуйста, подождите..."

#: gui/ScrRescuePisi.py:75
msgid "No connection available"
msgstr "Нет подключения к Интернету"

#: gui/ScrRescuePisi.py:116
#, python-format
msgid "Operation %(no)s : %(date)s - %(type)s"
msgstr "Операция %(no)s : %(date)s - %(type)s"

#: gui/ScrRescuePisi.py:170
#, python-format
msgid "Connecting to network %s..."
msgstr "Подключение к сети %s..."

#: gui/ScrRescuePisi.py:176 gui/ScrRescuePisi.py:177
msgid "Connection failed"
msgstr "Не удалось установить соединение"

#: gui/ScrRescuePisi.py:180 gui/ScrRescuePisi.py:181
msgid "Connected"
msgstr "Подключено"

#: gui/ScrDriveSelection.py:96
#, python-format
msgid "Device Path: %s"
msgstr ""

#: gui/ScrDriveSelection.py:112 gui/YaliWindow.py:381
#: storage/storageset.py:383 storage/storageset.py:459
msgid "Error"
msgstr "Ошибка"

#: gui/ScrDriveSelection.py:113
msgid "You must select at least one drive to be used for installation."
msgstr ""
"Вы должны выбрать хотя бы один из дисков, которые будут использоваться для "
"установки."

#: gui/partition_gui.py:34
#, python-format
msgid "Create Partition on %(path)s (%(model)s)"
msgstr "Создать раздел на %(path)s (%(model)s)"

#: gui/partition_gui.py:38
#, python-format
msgid "Edit Partition %s"
msgstr "Изменить раздел %s"

#: gui/partition_gui.py:40
msgid "Edit Partition"
msgstr "Изменить раздел"

#: gui/partition_gui.py:103
msgid "Error With Request"
msgstr "Ошибка с запросом"

#: gui/ScrUsers.py:127 gui/ScrUsers.py:359
msgid "Update"
msgstr "Обновление"

#: gui/ScrUsers.py:266
msgid "This user name is already taken, please choose another one."
msgstr "Это имя пользователя уже занято, пожалуйста, выберите другое."

#: gui/ScrUsers.py:270
msgid "The user name contains invalid characters."
msgstr "Имя пользователя содержит недопустимые символы."

#: gui/ScrUsers.py:273
msgid "The real name contains invalid characters."
msgstr "Настоящее имя содержит недопустимые символы."

#: gui/ScrUsers.py:281
msgid "User ID used before, choose another one!"
msgstr "Пользовательский ID уже использован, выберите другой!"

#: gui/ScrUsers.py:286 gui/ScrUsers.py:329 gui/ScrUsers.py:413
#: gui/Ui/setupuserswidget.py:510 gui/Ui/volumegroup.py:127
msgid "Add"
msgstr "Добавить"

#: gui/YaliWindow.py:90
msgid "Pixmaps resources file doesn't exists"
msgstr "Файл ресурсов растровых изображений не существует"

#: gui/YaliWindow.py:123
msgid "Style file doesn't exists"
msgstr "Файл стиля не существует"

#: gui/YaliWindow.py:130
msgid "Release file doesn't exists"
msgstr "Файл Release не существует"

#: gui/YaliWindow.py:138
msgid "Turn Off Computer"
msgstr "Выключить компьютер"

#: gui/YaliWindow.py:139
msgid "Restart Computer"
msgstr "Перезагрузить компьютер"

#: gui/YaliWindow.py:140
msgid "Restart YALI"
msgstr "Перезагрузить YALI"

#: gui/YaliWindow.py:161 gui/YaliWindow.py:178 sysutils.py:53
msgid "right"
msgstr "справа"

#: gui/YaliWindow.py:177 gui/YaliWindow.py:180 sysutils.py:52
msgid "left"
msgstr "слева"

#: gui/YaliWindow.py:181
msgid "Mouse Settings"
msgstr "Настройки мыши"

#: gui/YaliWindow.py:182
#, python-format
msgid "You just clicked the <b>%s</b> mouse button."
msgstr "Вы нажали на <b>%s</b> кнопку мыши"

#: gui/YaliWindow.py:183
#, python-format
msgid "Do you want to switch to the <b>%s</b> handed configuration?"
msgstr "Вы хотите переключиться на конфигурацию <b>%s</b> руки"

#: gui/YaliWindow.py:213
msgid "Are you sure you want to shut down your computer now?"
msgstr "Вы уверены, что вы хотите выключить компьютер сейчас?"

#: gui/YaliWindow.py:218
msgid "Are you sure you want to restart your computer now?"
msgstr "Вы уверены, что вы хотите перезагрузить компьютер сейчас?"

#: gui/YaliWindow.py:223
msgid "Are you sure you want to restart the YALI installer now?"
msgstr "Вы уверены, что хотите перезапустить программу установки YALI сейчас?"

#: gui/YaliWindow.py:244
msgid "Terminal"
msgstr "Терминал"

#: gui/YaliWindow.py:249
msgid "Tetris"
msgstr "Тетрис"

#: gui/YaliWindow.py:382
#, fuzzy, python-format
msgid "An error occurred when attempting to load screens:%s"
msgstr ""
"Произошла ошибка при попытке загрузить компонент интерфейса установки.\n"
"\n"
"className = %s.Widget"

#: gui/YaliWindow.py:417 gui/Ui/main.py:258
msgid "Release Notes"
msgstr "Информация о выпуске"

#: gui/YaliWindow.py:440
msgid "Release notes could not be loaded."
msgstr "Не удалось загрузить примечания к выпуску."

#: gui/ScrBootloader.py:99 gui/Ui/bootloaderwidget.py:145
msgid "The first sector of"
msgstr "Первый сектор"

#: gui/ScrSummary.py:47
msgid "<b><p>Are you sure you want to restart the computer?</p></b>"
msgstr "<b><p>Вы уверены, что вы хотите перезагрузить компьютер?</p></b>"

#: gui/ScrSummary.py:59 gui/ScrSummary.py:211 gui/ScrSummary.py:218
#: gui/Ui/main.py:262
msgid "Next"
msgstr "Вперед"

#: gui/ScrSummary.py:68
#, python-format
msgid "Installation starts in <b>%s</b> seconds"
msgstr "Установка начнется через <b>%s</b> секунд"

#: gui/ScrSummary.py:76
msgid "Start Installation"
msgstr "Начать установку"

#: gui/ScrSummary.py:78
msgid "Apply Settings"
msgstr "Применить настройки"

#: gui/ScrSummary.py:95
msgid "Keyboard Settings"
msgstr "Настройки клавиатуры"

#: gui/ScrSummary.py:97
#, python-format
msgid "Selected keyboard layout is <b>%s</b>"
msgstr "Выбранная раскладка клавиатуры <b>%s</b>"

#: gui/ScrSummary.py:103
msgid "Date/Time Settings"
msgstr "Установка даты/времени"

#: gui/ScrSummary.py:105
#, python-format
msgid "Selected TimeZone is <b>%s</b>"
msgstr "Выбранный часовой пояс <b>%s</b>"

#: gui/ScrSummary.py:111
msgid "User Settings"
msgstr "Настройки пользователя"

#: gui/ScrSummary.py:113
#, python-format
msgid "User %(username)s (<b>%(realname)s</b>) added."
msgstr "Пользователь %(username)s (<b>%(realname)s</b>) добавлен."

#: gui/ScrSummary.py:115
#, python-format
msgid ""
"User %(username)s (<b>%(realname)s</b>) added with <u>administrator "
"privileges</u>."
msgstr ""
"Пользователь %(username)s (<b>%(realname)s</b>) добавлен с правами "
"<u>суперпользователя</u>."

#: gui/ScrSummary.py:121
msgid "Hostname Settings"
msgstr "Настройка имени хоста"

#: gui/ScrSummary.py:123
#, python-format
msgid "Hostname is set as <b>%s</b>"
msgstr "Имя хоста задано как <b>%s</b>"

#: gui/ScrSummary.py:129
msgid "Partition Settings"
msgstr "Настройки раздела"

#: gui/ScrSummary.py:136
msgid "Automatic Partitioning selected."
msgstr "Автоматическое разбиение выбрано."

#: gui/ScrSummary.py:138
msgid "Use All Space"
msgstr "Использовать все пространство."

#: gui/ScrSummary.py:139
#, python-format
msgid ""
"Removes all partitions on the selected %s device(s). <b><u>This includes "
"partitions created by other operating systems.</u></b>"
msgstr ""
"Удаление всех разделов на выбранном %s устройстве(ах). <b><u>Включая "
"разделы, созданные другими операционными системами.</u></b>"

#: gui/ScrSummary.py:143
msgid "Replace Existing Linux System(s)"
msgstr "Заменить существующий Linux систему(ы)"

#: gui/ScrSummary.py:144
#, python-format
msgid ""
"Removes all Linux partitions on the selected%s device(s). This does not "
"remove other partitions you may have on your storage device(s) (such as VFAT "
"or FAT32)"
msgstr ""
"Удаление всех Linux-разделов на выбранном %s устройстве(ах). <b><u>Это не "
"включает в себя удаление других разделов, которые вы могли иметь на вашем "
"устройстве хранения данных (такие как VFAT или FAT32).</u></b>"

#: gui/ScrSummary.py:149 gui/Ui/autopartwidget.py:173
msgid "Use Free Space"
msgstr "Использовать свободное место"

#: gui/ScrSummary.py:150
#, python-format
msgid ""
"Retains your current data and partitions and uses only the unpartitioned "
"space on the selected %s device(s), assuming you have enough free space "
"available."
msgstr ""
"Сохраняет ваши данные и разделы и использует только неразмеченное "
"пространство на выбранном  %s устройстве(ах), считая, что свободного места "
"достаточно."

#: gui/ScrSummary.py:156
msgid "Manual Partitioning selected."
msgstr "Выбрана ручная разметка диска"

#: gui/ScrSummary.py:164
msgid "Bootloader Settings"
msgstr "Настройка загрузчика"

#: gui/ScrSummary.py:165
#, python-format
msgid "GRUB will be installed to <b>%s</b>."
msgstr "GRUB будет установлен в <b>%s</b>."

#: gui/ScrSummary.py:167
msgid "GRUB will not be installed."
msgstr "GRUB не будет установлен."

#: gui/ScrSummary.py:174
msgid "Package Installation Settings"
msgstr "Настройка установки пакетов"

#: gui/ScrSummary.py:175
#, fuzzy, python-format
msgid "Collection <b>%s</b> selected"
msgstr "Выбрана установка вручную ( %s )"

#: gui/ScrSummary.py:195
msgid "Confirm"
msgstr "Подтвердить"

#: gui/ScrSummary.py:196
#, fuzzy
msgid ""
"The partitioning options you have selected will now be written to disk. Any "
"data on deleted or reformatted partitions will be lost."
msgstr ""
"Параметры разметки, которые вы выбрали, сейчас будут записаны на диск.\n"
" Любые данные на удаляемом или форматируемом разделе будут потеряны."

#: gui/ScrSummary.py:201
msgid "Write Changes to Disk"
msgstr "Записать изменения на диск"

#: gui/ScrSummary.py:201
msgid "Go Back"
msgstr "Вернуться"

#: gui/ScrSummary.py:287
msgid "Proprietary Hardware Drivers"
msgstr "Проприетарные драйвера устройств"

#: gui/ScrSummary.py:288
#, fuzzy
msgid ""
"<qt>Proprietary drivers are available which may be required to utilize the "
"full capabilities of your video card. These drivers are developed by the "
"hardware manufacturer and not supported by Pisi Linux developers since their "
"source code is not publicly available.<br><br><b>Do you want to install and "
"use these proprietary drivers instead of the default drivers?</b></qt>"
msgstr ""
"<qt>Проприетарные драйвера вам могут понадобиться для использования всех "
"возможностей вашей видеокарты. Эти драйвера разработаны производителями "
"видеокарт и не поддерживаются разработчиками Pardus, поскольку их код закрыт."
"<br><br><b>Вы хотите установить и использовать эти драйвера вместо драйверов "
"по умолчанию?</b></qt>"

#: gui/Ui/manualpartwidget.py:193
msgid "Device"
msgstr ""

#: gui/Ui/manualpartwidget.py:194
#, fuzzy
msgid "Mount Point"
msgstr "Ошибка монтирования"

#: gui/Ui/manualpartwidget.py:195
msgid "Label"
msgstr ""

#: gui/Ui/manualpartwidget.py:196
msgid "Type"
msgstr ""

#: gui/Ui/manualpartwidget.py:198 gui/Ui/volumegroup.py:126
msgid "Size"
msgstr ""

#: gui/Ui/manualpartwidget.py:199
msgid "Create"
msgstr ""

#: gui/Ui/manualpartwidget.py:200 gui/Ui/setupuserswidget.py:511
#: gui/Ui/volumegroup.py:128
msgid "Edit"
msgstr ""

#: gui/Ui/manualpartwidget.py:201 gui/Ui/setupuserswidget.py:512
#: gui/Ui/volumegroup.py:129 gui/lvm_gui.py:552
#: storage/storageBackendHelpers.py:92 storage/storageBackendHelpers.py:296
msgid "Delete"
msgstr "Удалить"

#: gui/Ui/manualpartwidget.py:202
msgid "Reset"
msgstr ""

#: gui/Ui/raid.py:114
msgid "Raid Device:"
msgstr ""

#: gui/Ui/raid.py:115 gui/Ui/partition.py:152
msgid "Use:"
msgstr ""

#: gui/Ui/raid.py:116
#, fuzzy
msgid "Filesystem:"
msgstr "Файловая система:"

#: gui/Ui/raid.py:118 gui/Ui/partition.py:158 gui/Ui/logicalvolume.py:160
msgid "Migrate"
msgstr ""

#: gui/Ui/raid.py:119
msgid "Raid Level:"
msgstr ""

#: gui/Ui/raid.py:120
#, fuzzy
msgid "Raid Members:"
msgstr "Ошибочные члены Raid"

#: gui/Ui/raid.py:121
msgid "Number of spares:"
msgstr ""

#: gui/Ui/collectionitem.py:93
#, fuzzy
msgid "Form"
msgstr "Форматировать"

#: gui/Ui/partition.py:153 gui/Ui/partedit.py:174 gui/Ui/partitionedit.py:154
#: gui/Ui/logicalvolume.py:154
msgid "Size :"
msgstr ""

#: gui/Ui/partition.py:154 gui/Ui/partition.py:156 gui/Ui/partedit.py:175
#: gui/Ui/partitionshrinkwidget.py:102 gui/Ui/partresize.py:102
#: gui/Ui/partitionedit.py:155 gui/Ui/logicalvolume.py:155
#: gui/Ui/logicalvolume.py:157
msgid " MB"
msgstr ""

#: gui/Ui/partition.py:155 gui/Ui/partresize.py:104
#: gui/Ui/logicalvolume.py:156
msgid "Resize"
msgstr ""

#: gui/Ui/partition.py:159 gui/Ui/partedit.py:178 gui/Ui/partitionedit.py:156
#: gui/Ui/logicalvolume.py:158
#, fuzzy
msgid "File System :"
msgstr "Файловая система:"

#: gui/Ui/partition.py:160
#, fuzzy
msgid "Force to be a primary partition"
msgstr "Создать отформатированный раздел RAID"

#: gui/Ui/license.py:117
msgid ""
"<html><head/><body><p>Before installing Pisi Linux on your computer, please "
"read and accept the terms of free / open source licenses of the softwares "
"included in this installation medium. <a href=\"#\"><span style=\" text-"
"decoration: underline; color:#56a2c0;\">License texts</span></a></p></body></"
"html>"
msgstr ""

#: gui/Ui/license.py:118
msgid "I understand and accept the terms"
msgstr ""

#: gui/Ui/installprogress.py:76
msgid "Installing Package: "
msgstr ""

#: gui/Ui/rootpasswidget.py:142
msgid "Administrator password"
msgstr ""

#: gui/Ui/rootpasswidget.py:143
#, fuzzy
msgid "Confirm password"
msgstr "Подтвердить сброс"

#: gui/Ui/rootpasswidget.py:144
msgid "Computer Name"
msgstr ""

#: gui/Ui/main.py:257
msgid "Yali"
msgstr ""

#: gui/Ui/main.py:259
msgid "Previous Screen"
msgstr ""

#: gui/Ui/main.py:260
#, fuzzy
msgid "Back"
msgstr "Вернуться"

#: gui/Ui/main.py:261
msgid "Next Screen"
msgstr ""

#: gui/Ui/partedit.py:173 gui/Ui/partitionedit.py:153
#: gui/Ui/logicalvolume.py:153
msgid "Use :"
msgstr ""

#: gui/Ui/partedit.py:176
#, fuzzy
msgid "Format :"
msgstr "Форматировать"

#: gui/Ui/partedit.py:177
#, fuzzy
msgid "Yes, format this partition"
msgstr "LVM"

#: gui/Ui/partedit.py:179
#, fuzzy
msgid "Delete Partition"
msgstr "Изменить раздел"

#: gui/Ui/partedit.py:180
#, fuzzy
msgid "Resize Partition"
msgstr "Изменить раздел"

#: gui/Ui/partedit.py:181
msgid "Changes will be applied after Summary Screen."
msgstr ""

#: gui/Ui/partedit.py:182
msgid "Refresh Disks"
msgstr ""

#: gui/Ui/partedit.py:183
#, fuzzy
msgid "Reset All Changes"
msgstr "Использовать все пространство."

#: gui/Ui/partedit.py:184
msgid "Apply"
msgstr ""

#: gui/Ui/connectionlist.py:84
msgid ""
"To use PiSi History support, you may need to be online. You can use a "
"connection profile from the list below:"
msgstr ""

#: gui/Ui/connectionlist.py:85 gui/Ui/autopartquestion.py:92
#: gui/Ui/dialog.py:78
msgid "X"
msgstr ""

#: gui/Ui/connectionlist.py:86
#, fuzzy
msgid "Connect"
msgstr "Подключено"

#: gui/Ui/autopartwidget.py:169
#, fuzzy
msgid "Use All Disk"
msgstr "Использовать все пространство."

#: gui/Ui/autopartwidget.py:171
msgid "Shrink Current System"
msgstr ""

#: gui/Ui/autopartwidget.py:175
#, fuzzy
msgid "Manual Partitioning"
msgstr "Выбрана ручная разметка диска"

#: gui/Ui/partitionshrinkwidget.py:100 gui/Ui/partitionshrink.py:73
msgid ""
"Which partition would you like to shrink to make room for your installation?"
msgstr ""

#: gui/Ui/partitionshrinkwidget.py:101
msgid "Shrink partition to size (in MB) :"
msgstr ""

#: gui/Ui/partitionshrinkwidget.py:104
msgid "Shrink"
msgstr ""

#: gui/Ui/rescuepisiwidget.py:84
msgid "<b>Connection status:</b>"
msgstr ""

#: gui/Ui/rescuepisiwidget.py:85
msgid ""
"To use PiSi's history feature, you may need to be online. Select connection "
"profile."
msgstr ""

#: gui/Ui/rescuepisiwidget.py:86
msgid "Select Connection"
msgstr ""

#: gui/Ui/rescuepisiwidget.py:87
msgid "Select the operation to undo:"
msgstr ""

#: gui/Ui/autopartquestion.py:91
msgid ""
"There appears to be more than one resizeable partition. You can select a "
"partition from the list below or YALI will select one for you."
msgstr ""

#: gui/Ui/autopartquestion.py:93
msgid "Automatically choose the partition to resize (recommended)"
msgstr ""

#: gui/Ui/autopartquestion.py:94
msgid "I will choose the target"
msgstr ""

#: gui/Ui/autopartquestion.py:95
#, fuzzy
msgid "Use Selected"
msgstr "Использовать свободное место"

#: gui/Ui/checkcdwidget.py:210
msgid ""
"You can validate the integrity of the installation packages. Although this "
"is an optional step, we highly recommend to do the validation in order to "
"avoid any possible errors during the installation process."
msgstr ""

#: gui/Ui/checkcdwidget.py:211
#, fuzzy
msgid "Validation failed"
msgstr "Проверка %s"

#: gui/Ui/checkcdwidget.py:212
#, fuzzy
msgid "Validation succeeded"
msgstr "Проверка %s"

#: gui/Ui/partresize.py:99
msgid ""
"Changes will be applied when you click \"Resize\". To cancel the changes, "
"click <b>\"Cancel\".\n"
"<br><br>\n"
"Warning: A resizing operation may corrupt the partition, rendering the data "
"on it unreachable. Make sure that you have backed up the data. Note that "
"this operation cannot be undone.<b>"
msgstr ""

#: gui/Ui/partitionedit.py:157
#, fuzzy
msgid "Force to be primary partition"
msgstr "LVM"

#: gui/Ui/rescuewidget.py:125
msgid "Choose what do you want to do to repair your system"
msgstr ""

#: gui/Ui/rescuewidget.py:126
#, fuzzy
msgid "Reinstall bootloader"
msgstr "Установка загрузчика..."

#: gui/Ui/rescuewidget.py:127
#, fuzzy
msgid "Reset forgotten password"
msgstr "Настройка пароля администрация..."

#: gui/Ui/rescuewidget.py:128
msgid "Return back to a previous system state"
msgstr ""

#: gui/Ui/keyboardwidget.py:142
#, fuzzy
msgid "Keyboard Layouts"
msgstr "Настройки клавиатуры"

#: gui/Ui/keyboardwidget.py:143
msgid "Test Layout"
msgstr ""

#: gui/Ui/rescuepasswordwidget.py:136
msgid "Choose the user you want to reset the password:"
msgstr ""

#: gui/Ui/rescuepasswordwidget.py:137
msgid "Password:"
msgstr ""

#: gui/Ui/rescuepasswordwidget.py:138 gui/Ui/setupuserswidget.py:504
#, fuzzy
msgid "Confirm Password"
msgstr "Подтвердить сброс"

#: gui/Ui/rescuepasswordwidget.py:139
#, fuzzy
msgid "Reset Password"
msgstr "Настройка пароля администрация..."

#: gui/Ui/setupuserswidget.py:501
msgid "User Name"
msgstr ""

#: gui/Ui/setupuserswidget.py:502
msgid "Real Name"
msgstr ""

#: gui/Ui/setupuserswidget.py:503
msgid "Password"
msgstr ""

#: gui/Ui/setupuserswidget.py:505
msgid "Custom User ID"
msgstr ""

#: gui/Ui/setupuserswidget.py:506
msgid "Not recommended"
msgstr ""

#: gui/Ui/setupuserswidget.py:507
msgid "Bypass authorization attempts for this user"
msgstr ""

#: gui/Ui/setupuserswidget.py:508
msgid "Add user to administrator (wheel) group"
msgstr ""

#: gui/Ui/setupuserswidget.py:509
msgid "Select auto login user"
msgstr ""

#: gui/Ui/setupuserswidget.py:514
msgid "Advanced"
msgstr ""

#: gui/Ui/partitionshrink.py:74
msgid "Shrink partition to size (in MB)"
msgstr ""

#: gui/Ui/help.py:65
msgid "help content"
msgstr ""

#: gui/Ui/rescuegrubwidget.py:118
msgid "Select where to install the GRUB bootloader"
msgstr ""

#: gui/Ui/rescuegrubwidget.py:119
msgid "First bootable disk (recommended)"
msgstr ""

#: gui/Ui/rescuegrubwidget.py:120
msgid "Disk partition where Pisi Linux is installed"
msgstr ""

#: gui/Ui/rescuegrubwidget.py:121
msgid "Selected disk below"
msgstr ""

#: gui/Ui/datetimewidget.py:181
msgid "Date"
msgstr ""

#: gui/Ui/datetimewidget.py:182
msgid "Time"
msgstr ""

#: gui/Ui/datetimewidget.py:183
msgid "Timezone"
msgstr ""

#: gui/Ui/logicalvolume.py:152
#, fuzzy
msgid "Logical Volume:"
msgstr "Логический том"

#: gui/Ui/welcome.py:98
msgid ""
"Pisi Linux is installed and running successfully... \n"
"\n"
"Before you login to system, you need to create your first user and set the "
"system wide configurations. "
msgstr ""

#: gui/Ui/createdevicewidget.py:143
#, fuzzy
msgid "Standard Partition"
msgstr "Изменить раздел"

#: gui/Ui/createdevicewidget.py:151
#, fuzzy
msgid "Raid Member"
msgstr "Ошибочные члены Raid"

#: gui/Ui/createdevicewidget.py:153
#, fuzzy
msgid "Raid Array"
msgstr "Raid массивы"

#: gui/Ui/exception.py:71
msgid "Unhandled exception occured"
msgstr ""

#: gui/Ui/exception.py:72
msgid ""
"Please fill a bug report at http://bugs.pisilinux.org.tr/ attaching the "
"following backtrace."
msgstr ""

#: gui/Ui/exception.py:74
msgid "Show Backtrace"
msgstr ""

#: gui/Ui/volumegroup.py:119
#, fuzzy
msgid "Volume Group Name:"
msgstr "Группа томов"

#: gui/Ui/volumegroup.py:120
#, fuzzy
msgid "Physical Extends"
msgstr "Физический том"

#: gui/Ui/volumegroup.py:121
#, fuzzy
msgid "Physical Volumes to use:"
msgstr "Физический том"

#: gui/Ui/volumegroup.py:122
msgid "Total Space:"
msgstr ""

#: gui/Ui/volumegroup.py:123
#, fuzzy
msgid "Free Space:"
msgstr "Использовать свободное место"

#: gui/Ui/volumegroup.py:124
#, fuzzy
msgid "Logical Volumes"
msgstr "Логический том"

#: gui/Ui/volumegroup.py:125
#, fuzzy
msgid "Mountpoint"
msgstr "Точка монтирования уже используется"

#: gui/Ui/volumegroup.py:130
#, fuzzy
msgid "Used Space:"
msgstr "Использовать все пространство."

#: gui/Ui/bootloaderwidget.py:141
#, fuzzy
msgid "Advanced settings"
msgstr "Регулировка настроек времени"

#: gui/Ui/bootloaderwidget.py:142
#, fuzzy
msgid "Default settings"
msgstr "Установка даты/времени"

#: gui/Ui/bootloaderwidget.py:143
#, fuzzy
msgid "Don't install bootloader"
msgstr "Установка загрузчика..."

#: gui/Ui/bootloaderwidget.py:144
msgid "The partition where Pisi Linux is installed"
msgstr ""

#: gui/lvm_gui.py:48
msgid "Not enough physical volumes"
msgstr "Не достаточно физических томов"

#: gui/lvm_gui.py:49
msgid ""
"At least one unused physical volume partition is needed to\n"
"create an LVM Volume Group.\n"
"Create a partition or RAID array of type \"physical volume\n"
"(LVM)\" and then select the \"LVM\" option again."
msgstr ""
"По крайней мере один неиспользованный реальный том должен быть\n"
"создан в группе томов LVM.\n"
"Создать раздел или массив RAID типа \"реальный том\"\n"
"(LVM)\" и затем выбрать опцию \"LVM\" заново."

#: gui/lvm_gui.py:60
msgid "Make LVM Volume Group"
msgstr "Сделать группу томов LVM"

#: gui/lvm_gui.py:63
#, python-format
msgid "Edit LVM Volume Group: %s"
msgstr "Редактировать группу томов LVM: %s"

#: gui/lvm_gui.py:65
msgid "Edit LVM Volume Group"
msgstr "Редактировать группу томов LVM"

#: gui/lvm_gui.py:91
msgid "Invalid Volume Group Name"
msgstr "Неверное имя группы томов"

#: gui/lvm_gui.py:98
msgid "Name in use"
msgstr "Имя для использования"

#: gui/lvm_gui.py:99
#, python-format
msgid "The volume group name \"%s\" is already in use. Please pick another."
msgstr "Имя группы томов \"%s\" уже используется. Пожалуйста, выберите другое."

#: gui/lvm_gui.py:403
msgid ""
"The physical extent size cannot be changed because\n"
"otherwise the space required by the currently defined\n"
"logical volumes will be increased to more than the\n"
"available space."
msgstr ""
"Реальное место не может быть изменено, потому что\n"
"место, выделенное для текущих заданных\n"
"логических разделов будет увеличено больше,\n"
"чем имеется доступного объема."

#: gui/lvm_gui.py:411
msgid "Confirm Physical Extent Change"
msgstr "Подтвердите физическое изменение"

#: gui/lvm_gui.py:412
msgid ""
"This change in the value of the physical extent will\n"
"require the sizes of the current logical volume requests\n"
"to be rounded up in size to an integer multiple of\n"
"the physical extent.\n"
"\n"
"This change will take effect\n"
"immediately."
msgstr ""
"Это изменение физического объема тома потребует,\n"
"чтобы запросы размеров текущего логического тома\n"
"были округлены до целого множителя\n"
"физического объема.\n"
"\n"
"Это изменение вступит в силу\n"
"немедленно."

#: gui/lvm_gui.py:433
#, python-format
msgid ""
"The physical extent size cannot be changed because\n"
"the value selected (%(curpe)10.2f MB) is larger than the\n"
"smallest physical volume (%(maxpvsize)10.2f MB) in the volume group."
msgstr ""
"Физический объем не может быть изменен, потому что\n"
"выбранное значение (%(curpe)10.2f Mб) больше, чем\n"
"наименьший физический том (%(maxpvsize)10.2f Mб) в группе томов."

#: gui/lvm_gui.py:446
#, python-format
msgid ""
"The physical extent size cannot be changed because the value selected "
"(%(curpe)10.2f MB) is too large compared to the size of the smallest "
"physical volume (%(maxpvsize)10.2f MB) in the volume group."
msgstr ""
"Физический размер не может быть изменен, потому что выбранное значение "
"(%(curpe)10.2f Mб) слишком велико по сравнению с минимальным физическим "
"томом (%(maxpvsize)10.2f Mб) в группе томов."

#: gui/lvm_gui.py:458
msgid "Too small"
msgstr "Слишком маленький"

#: gui/lvm_gui.py:459
msgid ""
"This change in the value of the physical extent will waste substantial space "
"on one or more of the physical volumes in the volume group."
msgstr ""
"Это изменение значения физического объема приведет к существенным потерям "
"пространства на одном или нескольких томах в группе томов."

#: gui/lvm_gui.py:481
#, python-format
msgid ""
"The physical extent size cannot be changed because the resulting maximum "
"logical volume size (%10.2f MB) is smaller than one or more of their "
"currently defined logical volumes."
msgstr ""
"Физический объем не может быть изменен, потому что окончательный "
"максимальный размер логического тома (%10.2f Mб) меньше, чем один или более "
"текущих определенных логических томов."

#: gui/lvm_gui.py:502
msgid "No free slots"
msgstr "Нет свободных слотов"

#: gui/lvm_gui.py:503
#, python-format
msgid ""
"You cannot create more than %d logical volume \n"
"per volume group."
msgstr ""
"Вы не можете создать больше %d логических дисков \n"
"в дисковой группе."

#: gui/lvm_gui.py:510
msgid "No free space"
msgstr "Нет свободного места"

#: gui/lvm_gui.py:511
msgid ""
"There is no room left in the volume group to create new\n"
"logical volumes. To add a logical volume you must reduce\n"
"the size of one or more of the currently existing logical\n"
"volumes"
msgstr ""
"Не осталось места в группе томов для создания\n"
"логических томов. Для добавления логического тома\n"
"Вам необходимо уменьшить размер одного или\n"
"нескольких существующих логических томов"

#: gui/lvm_gui.py:548 storage/storageBackendHelpers.py:88
#: storage/storageBackendHelpers.py:295
msgid "Confirm Delete"
msgstr "Подтвердить удаление"

#: gui/lvm_gui.py:549
#, python-format
msgid ""
"Are you sure you want to delete the \n"
"logical volume \"%s\"?"
msgstr ""
"Вы действительно хотите удалить \n"
"логический диск \"%s\"?"

#: gui/lvm_gui.py:582
msgid "Make Logical Volume"
msgstr "Сделать логический раздел"

#: gui/lvm_gui.py:584
#, python-format
msgid "Edit Logical Volume: %s"
msgstr "Редактировать логический раздел: %s"

#: gui/lvm_gui.py:656
msgid "Illegal Logical Volume Name"
msgstr "Неверное имя логического диска"

#: gui/lvm_gui.py:668
msgid "Illegal logical volume name"
msgstr "Неверное имя логического диска"

#: gui/lvm_gui.py:669
#, python-format
msgid "The logical volume name \"%s\" is already in use. Please pick another."
msgstr ""
"Имя логического диска \"%s\" уже используется. Пожалуйста, выберите другое "
"имя."

#: gui/lvm_gui.py:683
msgid "Illegal size"
msgstr "Недопустимый размер"

#: gui/lvm_gui.py:684
msgid "The requested size as entered is not a valid number greater than 0."
msgstr "Требуемый размер, который вы ввели, не является числом большим 0."

#: gui/lvm_gui.py:697
#, python-format
msgid ""
"The current requested size (%(size)10.2f MB) is larger than the maximum "
"logical volume size (%(maxlv)10.2f MB). To increase this limit you can "
"create more Physical Volumes from unpartitioned disk space and add them to "
"this Volume Group."
msgstr ""
"Запрошенный размер (%(size)10.2f Mб) больше, чем максимальный размер "
"логического тома (%(maxlv)10.2f Mб). Для увеличения предела вы можете "
"создать больше Физических Томов из неразмеченного дискового пространства и "
"добавить их в Группу Томов."

#: gui/lvm_gui.py:719
#, python-format
msgid ""
"The size entered for this logical volume (%(size)d MB) combined with the "
"size of the other logical volume(s) exceeds the size of the volume group "
"(%(tmpvgsize)d MB). Please make the volume group larger or make the logical "
"volume smaller."
msgstr ""
"Указанный размер для этого логического тома (%(size)d Mб) вместе с размером "
"других логических томов превышает размер группы томов (%(tmpvgsize)d Mб). "
"Пожалуйста, расширьте группу томов или сделайте логический том меньше."

#: postinstall.py:124
#, python-format
msgid "Preparing home directory for %s..."
msgstr "Подготовка домашнего каталога для %s..."

#: util.py:527 util.py:570
msgid "Release file is missing"
msgstr "Файл Release отсутствует"

#: util.py:529 util.py:572
msgid "Release file is inconsistent"
msgstr "Файл Release не соответствует"

#: util.py:532 util.py:575
msgid "Invalid xml file"
msgstr "Неверный XML-файл"

#: localedata.py:20
msgid "Turkish"
msgstr "Turkish"

#: localedata.py:22
msgid "Turkish F"
msgstr "Turkish F"

#: localedata.py:22
msgid "Turkish Q"
msgstr "Turkish Q"

#: localedata.py:31
msgid "English"
msgstr "English"

#: localedata.py:42
msgid "English GB"
msgstr "English GB"

#: localedata.py:52
msgid "Greek"
msgstr "Greek"

#: localedata.py:62
msgid "Afrikaans"
msgstr "Afrikaans"

#: localedata.py:72
msgid "Vietnamese"
msgstr "Vietnamese"

#: localedata.py:82
msgid "Catalan"
msgstr "Catalan"

#: localedata.py:93
msgid "Italian"
msgstr "Italian"

#: localedata.py:104
msgid "Czech"
msgstr "Czech"

#: localedata.py:114
msgid "Welsh"
msgstr "Welsh"

#: localedata.py:124
msgid "Arabic"
msgstr "Arabic"

#: localedata.py:134
msgid "Estonian"
msgstr "Estonian"

#: localedata.py:144
msgid "Spanish"
msgstr "Spanish"

#: localedata.py:155
msgid "Russian"
msgstr "Russian"

#: localedata.py:166
msgid "Dutch"
msgstr "Dutch"

#: localedata.py:177
msgid "Portuguese"
msgstr "Portuguese"

#: localedata.py:188
msgid "Norwegian"
msgstr "Norwegian"

#: localedata.py:198
msgid "Icelandic"
msgstr "Icelandic"

#: localedata.py:208
msgid "Polish"
msgstr "Polish"

#: localedata.py:219
msgid "Belgium"
msgstr "Belgium"

#: localedata.py:229
msgid "French"
msgstr "French"

#: localedata.py:240
msgid "Bulgarian"
msgstr "Bulgarian"

#: localedata.py:250
msgid "Slovenian"
msgstr "Slovenian"

#: localedata.py:260
msgid "Croatian"
msgstr "Croatian"

#: localedata.py:270
msgid "German"
msgstr "German"

#: localedata.py:281
msgid "Danish"
msgstr "Danish"

#: localedata.py:291
msgid "Brazilian"
msgstr "Brazilian"

#: localedata.py:301
msgid "Finnish"
msgstr "Finnish"

#: localedata.py:311
msgid "Hungarian"
msgstr "Hungarian"

#: localedata.py:322
msgid "Japanese"
msgstr "Japanese"

#: localedata.py:332
msgid "Malayalam"
msgstr "Malayalam"

#: localedata.py:342
msgid "Swedish"
msgstr "Swedish"

#: localedata.py:353
msgid "Macedonian"
msgstr "Macedonian"

#: localedata.py:363
msgid "Slovak"
msgstr "Slovak"

#: localedata.py:373
msgid "Ukrainian"
msgstr "Ukrainian"

#: localedata.py:383
msgid "Serbian"
msgstr "Serbian"

#: storage/storageset.py:298 storage/storageset.py:311
msgid "Invalid mount point"
msgstr "Неверная точка монтирования"

#: storage/storageset.py:299
#, python-format
msgid ""
"An error occurred when trying to create %s.  Some element of this path is "
"not a directory. This is a fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"Возникла ошибка при попытке создания %s. Некоторые элементы этого пути не "
"являются директорией. Это критическая ошибка и установка не может быть "
"продолжена.\n"
"\n"
" Нажмите <Enter> для выхода из программы установки."

#: storage/storageset.py:312
#, python-format
msgid ""
"An error occurred when trying to create %(mountpoint)s: %(msg)s.  This is a "
"fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"Возникла ошибка при попытке создания %(mountpoint)s: %(msg)s. Это "
"критическая ошибка и установка не может быть продолжена.\n"
"\n"
"Нажмите <Enter> для выхода из программы установки."

#: storage/storageset.py:329 storage/storageset.py:350
msgid "Unable to mount filesystem"
msgstr "Не удалось смонтировать файловую систему"

#: storage/storageset.py:330
#, python-format
msgid ""
"An error occurred mounting device %(path)s as %(mountpoint)s.  You may "
"continue installation, but there may be problems."
msgstr ""
"Возникла ошибка при монтировании устройства %(path)s как %(mountpoint)s. Вы "
"можете продолжить установку, но могут возникнуть проблемы."

#: storage/storageset.py:336 storage/__init__.py:75
msgid "Exit installer"
msgstr "Выход из установки"

#: storage/storageset.py:351
#, python-format
msgid ""
"An error occurred mounting device %(path)s as %(mountpoint)s: %(msg)s. This "
"is a fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"Возникла ошибка при монтировании устройства %(path)s как %(mountpoint)s: "
"%(msg)s. Это критическая ошибка и установка не может быть продолжена.\n"
"\n"
"Нажмите <Enter> для выхода из программы установки."

#: storage/storageset.py:386
msgid "Skip"
msgstr "Пропустить"

#: storage/storageset.py:418
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"is an old-style Linux swap partition.  If you want to use this device for "
"swap space, you must reformat as a new-style Linux swap partition."
msgstr ""
"Устройство :\n"
"\n"
" %s\n"
"\n"
"является старым типом раздела подкачки Linux. Если вы хотите использовать "
"это устройство как раздел подкачки, вы должна отформатировать его как новый "
"тип раздела подкачки Linux."

#: storage/storageset.py:429
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"in your /etc/fstab file is currently in use as a software suspend device, "
"which means your system is hibernating. If you are performing a new install, "
"make sure the installer is set to format all swap devices."
msgstr ""
"Устройство подкачки:\n"
"\n"
" %s\n"
"\n"
"в вашем файле /etc/fstab на данный момент определено для спящего режима "
"системы. При выполнении новой установки, убедитесь, что установщик "
"отформатирует все устройства подкачки."

#: storage/storageset.py:442
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"does not contain a supported swap volume.  In order to continue "
"installation, you will need to format the device or skip it."
msgstr ""
"Устройство подкачки:\n"
"\n"
" %s\n"
"\n"
"не содержит поддерживаемых томов подкачки. Для того, чтобы продолжить "
"установку, вам нужно отформатировать устройство или пропустить его."

#: storage/storageset.py:453
#, fuzzy, python-format
msgid ""
"Error enabling swap device %(name)s: %(msg)s<br><br>This most likely means "
"this swap device has not been initialized.<br><br>Press OK to exit the "
"installer."
msgstr ""
"Ошибка включения устройства подкачки %(name)s: %(msg)s\n"
"\n"
"Это скорее всего означает, что это устройство подкачки не было "
"инициализировано.\n"
"\n"
"Нажмите OK для выхода из программы установки."

#: storage/storageset.py:501
msgid "You have not created a bootable partition."
msgstr "Вы не создали загрузочный раздел."

#: storage/storageset.py:505
msgid "Bootable partitions cannot be on a logical volume."
msgstr "Загрузочные разделы не могут располагаться на логическом томе. "

#: storage/storageset.py:509
#, fuzzy
msgid "Bootable partitions cannot be on a raid array."
msgstr "Загрузочные разделы не могут располагаться на логическом томе. "

#: storage/storageset.py:516
#, python-format
msgid "Bootable partitions cannot be on an %s filesystem."
msgstr "Загрузочные разделы не могут располагаться на %s файловой системы."

#: storage/storageset.py:620
msgid "Ignoring noauto entry"
msgstr ""

#: storage/storageset.py:678
#, python-format
msgid "Failed to resolve %s (%s) from fstab"
msgstr ""

#: storage/storageset.py:684
#, python-format
msgid "Unrecognized filesystem type for %s (%s)"
msgstr ""

#: storage/storageset.py:691
#, python-format
msgid "Scanned format (%s) differs from fstab format (%s)"
msgstr ""

#: storage/storageset.py:713
#, python-format
msgid "Cannot open %s for read"
msgstr ""

#: storage/storageset.py:743
#, python-format
msgid "Fstab entry %s is malformed: %s"
msgstr ""

#: storage/bootloader.py:210 storage/bootloader.py:211
#: storage/bootloader.py:213 storage/bootloader.py:214
#, python-format
msgid "%s"
msgstr "%s"

#: storage/bootloader.py:302 storage/bootloader.py:306
msgid "Windows"
msgstr "Windows"

#: storage/devicetree.py:1569
msgid "This partition is part of an inconsistent LVM Volume Group."
msgstr "Этот раздел является частью несовместимой группы томов LVM."

#: storage/library/lvm.py:265
#, python-format
msgid "vginfo failed for %s"
msgstr "vginfo не удалось для %s"

#: storage/library/lvm.py:291
#, python-format
msgid "lvs failed for %s"
msgstr "lvs не удалось для %s"

#: storage/formats/disklabel.py:28
#, fuzzy
msgid "partition table"
msgstr "Раздел"

#: storage/formats/raidmember.py:21
msgid "software RAID"
msgstr ""

#: storage/formats/physicalvolume.py:19
#, fuzzy
msgid "physical volume (LVM)"
msgstr "Физический том"

#: storage/formats/dmraidmember.py:19
msgid "dm-raid member device"
msgstr ""

#: storage/formats/filesystem.py:228
#, python-format
msgid "Unknown return code: %d."
msgstr "Неизвестный код возврата: %d."

#: storage/formats/filesystem.py:247
#, fuzzy, python-format
msgid "Migrating %s filesystem on %s"
msgstr "Изменение размеров файловой системы на %s"

#: storage/formats/filesystem.py:299
#, python-format
msgid "Creating %(type)s filesystem on %(device)s"
msgstr "Создание %(type)s файловой системы на %(device)s"

#: storage/formats/filesystem.py:347
#, python-format
msgid "Resizing filesystem on %s"
msgstr "Изменение размеров файловой системы на %s"

#: storage/formats/filesystem.py:378
#, python-format
msgid "Checking filesystem on %s"
msgstr "Проверка файловой системы на %s"

#: storage/formats/filesystem.py:387
#, python-format
msgid "%(type)s filesystem check failure on %(device)s: "
msgstr "Проверка файловой системы %(type)s не удалась на %(device)s: "

#: storage/formats/filesystem.py:391
#, fuzzy
msgid ""
"Errors like this usually mean there is a problem with the filesystem that "
"will require user interaction to repair. Restart installation after you have "
"corrected the problems on the filesystem."
msgstr ""
"Ошибки как эти, обычно означают, что существует проблема с файловой "
"системой, что потребует вмешательства пользователя для ремонта. "
"Перезапустите установку после устранения проблем с файловой системой."

#: storage/formats/filesystem.py:397
msgid "Unrecoverable Error"
msgstr "Неустранимая ошибка"

#: storage/formats/filesystem.py:691
msgid "File system errors left uncorrected."
msgstr "Ошибки файловой системы остались неисправленными."

#: storage/formats/filesystem.py:692
msgid "Operational error."
msgstr "Оперативная ошибка."

#: storage/formats/filesystem.py:693
msgid "Usage or syntax error."
msgstr "Ошибка использования или синтаксиса."

#: storage/formats/filesystem.py:694
msgid "e2fsck cancelled by user request."
msgstr "e2fsck завершена пользователем."

#: storage/formats/filesystem.py:695
msgid "Shared library error."
msgstr "Ошибка общей библиотеки."

#: storage/formats/filesystem.py:855
msgid ""
"Recoverable errors have been detected or dosfsck has discovered an internal "
"inconsistency."
msgstr ""
"Обнаружены некритические ошибки или dosfsck обнаружила внутреннюю "
"противоречивость."

#: storage/formats/filesystem.py:857
msgid "Usage error."
msgstr "Ошибка использования."

#: storage/partitioning.py:703
msgid "requested size exceeds maximum allowed"
msgstr ""

#: storage/partitioning.py:911
#, python-format
msgid "partition is too %(problem)s for %(format)s formatting"
msgstr ""

#: storage/partitioning.py:1051
msgid "not enough free space on disks"
msgstr "не хватает свободного места на дисках"

#: storage/partitioning.py:1072
msgid "not enough free space after creating extended partition"
msgstr ""

#: storage/partitioning.py:1248
#, fuzzy
msgid "not enough space for LVM requests"
msgstr "не хватает места для запросов LVM"

#: storage/partitioning.py:1645
msgid ""
"Could not find enough free space for automatic\n"
"partitioning, please use another partitioning method."
msgstr ""
"Не удалось найти достаточно свободного места для автоматической разметки, "
"пожалуйста, используйте другой метод разметки диска."

#: storage/partitioning.py:1669
msgid "Warnings During Automatic Partitioning"
msgstr "Предупреждения во время автоматической разметки"

#: storage/partitioning.py:1670
#, python-format
msgid ""
"Following warnings occurred during automatic partitioning:\n"
"\n"
"%s"
msgstr ""
"Возникли следующие предупреждения в процессе автоматической разметки:\n"
"\n"
"%s"

#: storage/partitioning.py:1679 storage/partitioning.py:1699
msgid ""
"\n"
"\n"
"Press 'OK' to exit the installer."
msgstr ""
"\n"
"\n"
"Нажмите 'OK' для выхода из программы установки."

#: storage/partitioning.py:1681
#, python-format
msgid ""
"Could not allocate requested partitions: \n"
"%(msg)s.%(extra)s"
msgstr ""
"Не удалось разместить требуемые разделы:\n"
"\n"
"%(msg)s.%(extra)s"

#: storage/partitioning.py:1700
msgid "Automatic Partitioning Errors"
msgstr "Ошибки автоматической разметки дисков"

#: storage/partitioning.py:1701
#, python-format
msgid ""
"The following errors occurred with your partitioning:\n"
"\n"
"%(errortxt)s\n"
"\n"
"This can happen if there is not enough space on your hard drive(s) for the "
"installation. %(extra)s"
msgstr ""
"Следующие ошибки произошли в процессе разметки:\n"
"\n"
"%(errortxt)s\n"
"\n"
"Это могло произойти, если не хватает места на жестком диске(ах) для "
"установки. %(extra)s"

#: storage/devices/filedevice.py:93
#, python-format
msgid "Creating file %s"
msgstr "Создание файла %s"

#: storage/devices/partition.py:369 storage/devices/raidarray.py:422
#: storage/devices/logicalvolume.py:217 storage/devices/volumegroup.py:245
#, python-format
msgid "Creating device %s"
msgstr "Создание устройства %s"

#: storage/devices/volumegroup.py:217
msgid "device has not been created"
msgstr ""

#: storage/devices/volumegroup.py:223
msgid "cannot activate VG with missing PV(s)"
msgstr ""

#: storage/storageBackendHelpers.py:18
msgid "Please enter a volume group name."
msgstr "Пожалуйста, введите имя группы томов."

#: storage/storageBackendHelpers.py:22
msgid "Volume Group Names must be less than 128 characters"
msgstr "Имя группы томов не должно быть длиннее 128 символов."

#: storage/storageBackendHelpers.py:25
#, python-format
msgid "Error - the volume group name %s is not valid."
msgstr "Ошибка - имя группы томов %s не правильное."

#: storage/storageBackendHelpers.py:30
msgid ""
"Error - the volume group name contains illegal characters or spaces.  "
"Acceptable characters are letters, digits, '.' or '_'."
msgstr ""
"Ошибка - имя группы томов содержит недопустимые символы или пробелы. "
"Допустимые символы - буквы, цифры, '.' или '_'."

#: storage/storageBackendHelpers.py:40
msgid "Please enter a logical volume name."
msgstr "Пожалуйста, введите имя логического тома."

#: storage/storageBackendHelpers.py:44
msgid "Logical Volume Names must be less than 128 characters"
msgstr "Имя логического тома не должно быть длиннее 128 символов."

#: storage/storageBackendHelpers.py:48
#, python-format
msgid "Error - the logical volume name %s is not valid."
msgstr "Ошибка - имя логического тома %s не верно задано."

#: storage/storageBackendHelpers.py:54
msgid ""
"Error - the logical volume name contains illegal characters or spaces.  "
"Acceptable characters are letters, digits, '.' or '_'."
msgstr ""
"Ошибка - имя логического тома содержит недопустимые символы или пробелы. "
"Допустимые символы - буквы, цифры, '.' или '_'."

#: storage/storageBackendHelpers.py:68
msgid "Format as Swap?"
msgstr "Форматировать как раздел подкачки (swap)?"

#: storage/storageBackendHelpers.py:69
#, python-format
msgid ""
"%s has a partition type of 0x82 (Linux swap) but does not appear to be "
"formatted as a Linux swap partition.\n"
"\n"
"Would you like to format this partition as a swap partition?"
msgstr ""
"%s имеет тип 0x82 (Linux swap), но по всей видимости, не был отформатирован "
"как раздел подкачки Linux.\n"
"\n"
"Вы хотите отформатировать этот раздел как раздел подкачки?"

#: storage/storageBackendHelpers.py:89
#, python-format
msgid "You are about to delete all partitions on the device '%s'."
msgstr "Вы собираетесь удалить все разделы на устройстве '%s'."

#: storage/storageBackendHelpers.py:139
msgid "Notice"
msgstr "Внимание"

#: storage/storageBackendHelpers.py:140
#, python-format
msgid ""
"The following partitions were not deleted because they are in use:\n"
"\n"
"%s"
msgstr ""
"Следующие разделы не были удалены, потому что они используются в данный "
"момент:\n"
"\n"
"%s"

#: storage/storageBackendHelpers.py:154 storage/storageBackendHelpers.py:161
msgid "Unable To Delete"
msgstr "Удаление не удалось"

#: storage/storageBackendHelpers.py:155
msgid "You must first select a partition to delete."
msgstr "Сначала вы должны выбрать раздел для удаления."

#: storage/storageBackendHelpers.py:200
#, python-format
msgid ""
"The mount point %s is invalid.  Mount points must start with '/' and cannot "
"end with '/', and must contain printable characters and no spaces."
msgstr ""
"Точка монтирования %s не правильная. Точки монтирования должны начинаться с "
"'/' и не должны заканчиваться на '/', содержать только печатные символы, без "
"пробелов."

#: storage/storageBackendHelpers.py:209
msgid "Error with Partitioning"
msgstr "Ошибка разметки"

#: storage/storageBackendHelpers.py:210
#, python-format
msgid ""
"The following critical errors exist with your requested partitioning scheme. "
"These errors must be corrected prior to continuing with your install of "
"%(productName)s.\n"
"\n"
"%(errorstr)s"
msgstr ""
"Следующие критические ошибки существует в вашей схеме разметки. Эти ошибки "
"должны быть исправлены для продолжения установки %(productName)s.\n"
"\n"
"%(errorstr)s"

#: storage/storageBackendHelpers.py:226
msgid "Partitioning Warning"
msgstr "Предупреждения разметки дисков"

#: storage/storageBackendHelpers.py:227
#, python-format
msgid ""
"The following warnings exist with your requested partition scheme.\n"
"\n"
"%s\n"
"\n"
"Would you like to continue with your requested partitioning scheme?"
msgstr ""
"Схема разметки содержит предупреждения. \n"
"\n"
"%s\n"
"\n"
"Вы хотите продолжить с данной схемой разметки?"

#: storage/storageBackendHelpers.py:242
msgid ""
"The following pre-existing partitions have been selected to be formatted, "
"destroying all data."
msgstr ""
"Следующие уже существующие устройства были выбраны для форматирования. При "
"этом будут уничтожены все данные."

#: storage/storageBackendHelpers.py:245
msgid ""
"Select 'Yes' to continue and format these partitions, or 'No' to go back and "
"change these settings."
msgstr ""
"Выберите 'Да' для продолжения и форматирования этих разделов, или 'Нет' для "
"возврата назад и изменения этих параметров."

#: storage/storageBackendHelpers.py:251
msgid "Format Warning"
msgstr "Предупреждения форматирования"

#: storage/storageBackendHelpers.py:279
#, python-format
msgid ""
"You are about to delete the volume group \"%s\".\n"
"\n"
"ALL logical volumes in this volume group will be lost!"
msgstr ""
"Вы собираетесь удалить группу томов \"%s\".\n"
"\n"
"ВСЕ логические диски в этой группе томов будут потеряны!"

#: storage/storageBackendHelpers.py:283
#, python-format
msgid "You are about to delete the logical volume \"%s\"."
msgstr "Вы собираетесь удалить логический том \"%s\"."

#: storage/storageBackendHelpers.py:286
msgid "You are about to delete a RAID device."
msgstr "Вы собираетесь удалить RAID устройство."

#: storage/storageBackendHelpers.py:288
#, python-format
msgid "You are about to delete the %s partition."
msgstr "Вы собираетесь удалить %s раздел."

#: storage/storageBackendHelpers.py:292
#, python-format
msgid "You are about to delete the %(type)s %(name)s"
msgstr "Вы собираетесь удалить %(type)s %(name)s"

#: storage/storageBackendHelpers.py:302
msgid "Confirm Reset"
msgstr "Подтвердить сброс"

#: storage/storageBackendHelpers.py:302
msgid ""
"Are you sure you want to reset the partition table to its original state?"
msgstr ""
"Вы действительно хотите сбросить таблицу разделов в исходное состояние?"

#: storage/storageBackendHelpers.py:310
msgid ""
"You have chosen to use a pre-existing partition for this\n"
"installation without formatting it. We recommend that you format\n"
"this partition to make sure files from a previous operating system\n"
"installation do not cause problems with this installation of Linux.\n"
"However, if this partition contains files that you need to keep,\n"
"such as home directories, then continue without formatting this\n"
"npartition."
msgstr ""
"Вы решили использовать уже существующий раздел для установки без\n"
"форматирования. Мы рекомендуем Вам отформатировать этот раздел, чтобы быть "
"уверенным в том, что файлы из\n"
"предыдущей установки операционной системы не вызовут проблем с текущей "
"установкой Linux.\n"
"Однако, если этот раздел содержит файлы, которые необходимо сохранить,\n"
"такие как домашние директории, тогда продолжайте без форматирования этого\n"
"раздела."

#: storage/storageBackendHelpers.py:318
msgid "Format?"
msgstr "Форматировать?"

#: storage/storageBackendHelpers.py:319
msgid "Do Not Format"
msgstr "Не форматировать"

#: storage/storageBackendHelpers.py:326
#, python-format
msgid ""
"Partitions of type '%s' must be constrained to a\n"
"single drive.  To do this, select the drive in the 'Allowable Drives'\n"
"checklist."
msgstr ""
"Разделы типа '%s' должны быть объединены в один диск. Это производится\n"
"путем выбора диска в списке 'Доступные диски'."

#: storage/storageBackendHelpers.py:331 storage/storageBackendHelpers.py:356
#: storage/storageBackendHelpers.py:376
#, fuzzy
msgid "Storage Device Warning"
msgstr "Определение устройств хранения данных..."

#: storage/storageBackendHelpers.py:332
#, python-format
msgid ""
"The following storage device may contain recoverable data:<br><br>"
"%(description)s (%(size)s MB)<br><br>This device may have data that will be "
"<b>permanently lost</b> if you include it in this installation.<br><br>If "
"you don't care about any data on this device, choose <b>Reset device</b> to "
"include it in the installation process.<br><br>If you don't want the device "
"to be permanently erased, select <b>Ignore device</b> and the device will "
"not be considered as a possible installation target."
msgstr ""

#: storage/storageBackendHelpers.py:343
msgid "Reset device"
msgstr ""

#: storage/storageBackendHelpers.py:343
msgid "Ignore device"
msgstr "Игнорировать устройство"

#: storage/storageBackendHelpers.py:357
#, fuzzy, python-format
msgid ""
"There is inconsistent LVM data on %(msg)s. You can reinitialize all related "
"PVs (%(pvs)s) which will erase the LVM metadata, or ignore which will "
"preserve thecontents.This action may also be applied to all other PVs with "
"inconsistent metadata."
msgstr ""
"Ошибка обработки LVM.\n"
"Есть некорректная информация LVM на %(msg)s. Вы\n"
"можете реинициализировать все связанные PV (%(pvs)s), что сотрет метаданные "
"LVM или игнорировать, что сохранит\n"
"содержимое. Это действие может быть также проделано с другими PV c\n"
"некорректными метаданными."

#: storage/storageBackendHelpers.py:364
msgid "Yes, re-initialize"
msgstr "Да, ре-инициализировать"

#: storage/storageBackendHelpers.py:364
msgid "No, protect data"
msgstr "Нет, защитить данные"

#: storage/storageBackendHelpers.py:377
#, python-format
msgid ""
"Disk contains %(members_count)s BIOS RAID metadata, but is not part of any "
"recognized BIOS RAID sets. Ignoring disk %(members)s."
msgstr ""
"Диск содержит %(members_count)s BIOS RAID метаданных, но они не являются "
"частью какого-либо известного BIOS RAID набора. Пропуск диска %(members)s."

#: storage/__init__.py:48
#, python-format
msgid "Failed teardownAll in storage.complete with error:%s"
msgstr "Неудачный teardownAll в storage.complete с ошибкой:%s"

#: storage/__init__.py:57
#, fuzzy
msgid "Storage Device Error"
msgstr "Определение устройств хранения данных..."

#: storage/__init__.py:58
#, fuzzy, python-format
msgid "There was an error encountered while partitioning on device %s."
msgstr "Возникла ошибка в процессе изменения размеры устройства %s."

#: storage/__init__.py:63
msgid "Storage Format Error"
msgstr ""

#: storage/__init__.py:64
#, fuzzy, python-format
msgid "There was an error encountered while formatting on device %s."
msgstr "Возникла ошибка в процессе изменения размеры устройства %s."

#: storage/__init__.py:75
#, fuzzy
msgid "Ignore"
msgstr "Игнорировать все"

#: storage/__init__.py:158 storage/__init__.py:168
#, fuzzy
msgid "Dirty File Systems"
msgstr "Файловая система:"

#: storage/__init__.py:159
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Please boot your Linux installation, let the file systems be "
"checked and shut down cleanly to rescue.\n"
"%s"
msgstr ""

#: storage/__init__.py:169
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Would you like to mount them anyway?\n"
"%s"
msgstr ""

#: storage/__init__.py:336
msgid "This partition is holding the data for the hard drive install."
msgstr "Этот раздел содержит данные по установке на жесткий диск."

#: storage/__init__.py:339
msgid "You cannot delete a partition of a LDL formatted DASD."
msgstr "Вы не можете удалять раздел LDL, отформатированный DASD."

#: storage/__init__.py:345
#, python-format
msgid ""
"This device is part of the RAID device %s, you have to edit or remove the "
"raid partition appropriately!"
msgstr ""
"Этот раздел является частью RAID устройства %s, вы должны редактировать или "
"удалять RAID-разделы надлежащим для этого образом!"

#: storage/__init__.py:350
msgid "This device is part of a RAID device."
msgstr "Это устройство является частью устройства RAID."

#: storage/__init__.py:355
#, python-format
msgid "This device is part of the LVM volume group '%s'."
msgstr "Это устройство является частью дисковой группы LVM '%s'."

#: storage/__init__.py:358
msgid "This device is part of a LVM volume group."
msgstr "Это устройство является частью дисковой группы LVM."

#: storage/__init__.py:367
msgid ""
"This device is an extended partition which contains logical partitions that "
"cannot be deleted:\n"
"\n"
msgstr ""
"Это устройство представляет собой расширенный раздел, содержащий логические "
"разделы, которые не могут быть удалены:\n"
"\n"

#: storage/__init__.py:873
#, fuzzy, python-format
msgid ""
"You have not defined a root partition (/),which is required for installation "
"of %s to continue."
msgstr ""
"Вы не задали корневой раздел (/), который необходим для продолжения "
"установки %s."

#: storage/__init__.py:878
#, fuzzy, python-format
msgid ""
"Your / partition is less than %(min)s MB which is lower than recommended for "
"a normal %(productName)s install."
msgstr ""
"Ваш / раздел меньше чем %(min)s MB, что ниже рекомендуемого размера для "
"нормальной установки %(productName)s ."

#: storage/__init__.py:884
#, fuzzy, python-format
msgid ""
"Your %(mount)s partition is less than %(size)s megabytes which is lower than "
"recommended for a normal %(productName)s install."
msgstr ""
"Ваш %(mount)s раздел меньше чем %(size)s MB, что ниже рекомендуемого размера "
"для нормальной установки %(productName)s ."

#: storage/__init__.py:893
#, fuzzy
msgid ""
"You have not specified a swap partition. Due to the amount of memory "
"present, a swap partition is required to complete installation."
msgstr ""
"Вы не задали раздел подкачки. В связи с представленным объемом оперативной "
"памяти, раздел подкачки необходим для успешного завершения установки."

#: storage/__init__.py:896
#, fuzzy
msgid ""
"You have not specified a swap partition. Although not strictly required in "
"all cases, it will significantly improve performancefor most installations."
msgstr ""
"Вы не задали раздел подкачки. Хоть это и не является крайне обязательным "
"шагом во всех случаях, но это позволит значительно повысить "
"производительность для большинства установок."

#: storage/__init__.py:902
#, python-format
msgid ""
"This mount point is invalid. The %s directory must be on the / file system."
msgstr ""
"Это точка монтирования не верна. Директория %s должна быть на файловой "
"системе / ."

#: storage/__init__.py:905
#, python-format
msgid "The mount point %s must be on a linux file system."
msgstr "Точка монтирования %s должна располагаться в файловой системе Linux."

#: storage/__init__.py:925
msgid "No Drives Found"
msgstr "Диски не обнаружены"

#: storage/__init__.py:926
msgid ""
"An error has occurred - no valid devices were found on\n"
"which to create new file systems. Please check your\n"
"hardware for the cause of this problem."
msgstr ""
"Произошла ошибка - не были найдены правильные устройства для создания\n"
"новых файловых систем. Проверьте оборудование для поиска причин\n"
"этой проблемы."
